plugins/database/src/db_plugin.cpp

```diff
@@ -15403,38 +15403,32 @@ auto db_delay_rule_unlock(irods::plugin_context& _ctx, const char* _rule_ids) ->
     }
 } // db_delay_rule_unlock
 
-auto db_update_replica_access_time(irods::plugin_context& _ctx, const char* _input, [[maybe_unused]] char** _output) -> irods::error
+auto db_update_replica_access_time(irods::plugin_context& _ctx, const char* _json_input, [[maybe_unused]] char** _output) -> irods::error
 {
     if (const auto ret = _ctx.valid(); !ret.ok()) {
         return PASS(ret);
     }
 
-    if (!_input) {
-        log_db::error("{}: JSON input cannot be a null pointer.", __func__);
-        return ERROR(SYS_INTERNAL_NULL_INPUT_ERR, "JSON input cannot be a null pointer.");
+    if (!_json_input || !_output) {
+        log_db::error("{}: Received one or more null pointers.", __func__);
+        return ERROR(SYS_INTERNAL_NULL_INPUT_ERR, "Received one or more null pointers.");
     }
 
     // The output pointer is not used because the ODBC API isn't guaranteed to tell us
-    // which SQL statement failed.
+    // which SQL statement failed. However, it is made available to future proof the APIs
+    // supporting access time.
 #if 0
     if (!_output) {
-        log_db::error("{}: Rule ID list cannot be a null pointer.", __func__);
+        log_db::error("{}:  list cannot be a null pointer.", __func__);
         return ERROR(SYS_INTERNAL_NULL_INPUT_ERR, "Rule ID list cannot be a null pointer.");
```

Comment:
this error now inconsistent with the one logged.

Reply:
Updated.